jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassUtilities.java

```diff
@@ -177,7 +176,7 @@
 	 * @throws		IllegalStateException
 	 * 					If shared classes is disabled for this JVM (that is -Xshareclasses:none is present).
 	 * @throws		IllegalArgumentException
-	 * 					If <code>cacheType<code> is not a valid value.
+	 * 					If <code>cacheType</code> is not a valid value.
```

Comment:
We should prefer `{@code ...}` over `<code>...</code>`.

Reply:
I'm not keen to change all the places that use `<code>` just to continue to compile jdk17. I suggest this should remain as `<code>` to be consistent with the rest of the file, and changing to use `{@code ...}` should be done everywhere in a different PR.

Reply:
Actually, using `{@code` (where the closing `}`) is not on the same line breaks jdk11. :-(
We'll need to escape things like `@` (as `{@literal @}` or `&#x40;`), `<` (as `&lt;`), etc.